pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_intervention_sweep.py

```diff
@@ -129,8 +129,15 @@ def test___call__(self):
                  if isinstance(key, CaseIsolation)][0]])
         self.assertIsNone(self.person_symp.isolation_start_time)
 
-        # The second place closure is active after its starting time
         self.interventionsweep(time=200)
+        # The first place closure is inactive after
```

Comment:
Am I misreading this test? It looks like you are asserting that the active status is true, but the comment saying you are checking that the first intervention is now inactive, so I would have expected `assertFalse` here

Reply:
Changed now